### ISSUE TITLE
deep link to tag, search by URL

### ISSUE THREAD

**Issue by pieper** [innolitics/dicom-standard#42]
**Is your feature request related to a problem? Please describe.**

We want to help people understand documents that include descriptions of dicom headers by linking out to information about a tag.

**Proposed feature**

It would be great if we could deep link to a tag search in your browser.

Currently I can deeplink to a tag if I know the IOD it belongs to:

https://dicom.innolitics.com/ciods/ct-image/general-study/00080020

I'd like to be able to include a deep line like:

https://dicom.innolitics.com/search/00080020

that would be the the same as typing 00080020 into the search tab.

But from what I can see the search interface is currently using an HTTP POST, and there's no way to embed the search term in the URL.

**Potential alternatives**

The search term could be in url parameters instead of being part of the URL itself.

**Additional context (e.g. screenshots)**

**Comment by johndgiese** [comment]
Thanks @pieper this is a great idea! I'll be sure to include it high on the feature list for our next set of improvements (I'm not sure at this point when that will be, but I'll let you know once it goes live!)

I hope you are doing well!

**Comment by johndgiese** [comment]
Ahh, that makes a lot of sense; we would love to have Slicer linked with the DICOM Standard Browser, and so we would like to do our part to make this happen.

As you know, the DICOM standard browser's URL structure ties the tags to their location within a CIOD's representation as a DICOM Data Set. Thus, given just a tag, there is not a single canonical page to look at. (I suspect this is why you were thinking of using the search page.)

Would it be helpful if we created some pages which are specifically tied to a tag?

E.g., in addition to the page for 00100020 within the CT Image, which has this URL

https://dicom.innolitics.com/ciods/ct-image/patient/00100020

we could add a new page as such

https://dicom.innolitics.com/tags/00100020

?

I think your suggestion to make it possible to deep link to a tag would be worthwhile regardless, but maybe adding tag-sepcific pages would be helpful for this use case.

**Comment by pieper** [comment]
Yes, a tag-specific page would be perfect!  We just want to get people into a useful spot so they can better understand their data.
